collectors/proc.plugin/multi_metadata.yaml

```diff
@@ -597,20 +605,23 @@ modules:
       plugin_name: proc.plugin
       module_name: /proc/loadavg
       monitored_instance:
-        name: proc /proc/loadavg
+        name: Load
         link: ''
-        categories: []
-        icon_filename: ''
+        categories:
+         - data-collection.linux-systems.system-metrics
+        icon_filename: 'syslog.png'
       related_resources:
         integrations:
           list: []
       info_provided_to_referring_integrations:
         description: ''
-      keywords: []
+      keywords:
+       - load
+       - load average
       most_popular: false
     overview:
       data_collection:
-        metrics_description: ''
+        metrics_description: "This integration provides the number of service requests that have been sent to the system's various interrupt lines."
```

Comment:
Wrong section, load average has nothing to do with interrupts.

Reply:
that's true, must've slipped from my awareness, and inputted the wrong thing.